src/schemes/fluid/surface_tension.jl

```diff
@@ -146,3 +130,69 @@ function calc_normal_akinci!(system, neighbor_system, surface_tension, u_system,
     # Normal not needed
     return system
 end
+
+@inline function surface_tension_force(surface_tension_a::CohesionForceAkinci,
+                                       surface_tension_b::CohesionForceAkinci,
+                                       particle_system::FluidSystem,
+                                       neighbor_system::FluidSystem, particle, neighbor,
+                                       pos_diff, distance)
+    (; smoothing_length) = particle_system
+    # No cohesion with oneself
+    distance < sqrt(eps()) && return zero(pos_diff)
+    m_b = hydrodynamic_mass(neighbor_system, neighbor)
+    support_radius = compact_support(smoothing_kernel, smoothing_length)
+
+    return cohesion_force_akinci(surface_tension_a, support_radius, m_b, pos_diff, distance)
+end
+
+@inline function surface_tension_force(surface_tension_a::SurfaceTensionAkinci,
+                                       surface_tension_b::SurfaceTensionAkinci,
+                                       particle_system::FluidSystem,
+                                       neighbor_system::FluidSystem, particle, neighbor,
+                                       pos_diff, distance)
+    (; smoothing_length, smoothing_kernel) = particle_system
+    (; surface_tension_coefficient) = surface_tension_a
+
+    # No surface tension with oneself
+    distance < sqrt(eps()) && return zero(pos_diff)
```

Comment:
```suggestion
    distance < sqrt(eps()) && return zero(pos_diff)
    
```
A little easier to read IMHO.